pkg/util/config/config.go

```diff
@@ -199,7 +199,7 @@ type Config struct {
 
 // MustMarshal marshals the config or panics
 func (c Config) MustMarshal() string {
-	b, err := json.MarshalIndent(c, "", "\t")
+	b, err := json.MarshalIndent(c, "", "   ")
```

Comment:
why ? imo `\t` is more readable

Reply:
\t gets serialized to \t i think so makes it not readable, you can try after PR to change it.